<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="pcd_filename" default="cloud"/>
    <arg name="image_filename" default="image"/>
    <arg name="auto_mode" default="false"/>
    <arg name="start_flag_cam" default="/imgCaptureFinished"/>
    <arg name="start_flag_lidar" default="/pcdTransferFinished"/>
    
    <group ns="lidar">
        <node pkg="manual_calibration" name="selectPointLidar" type="selectPointLidar" output="screen">
            <param name="pcd_filepath" value="$(find manual_calibration)/data/pcdFiles/$(arg pcd_filename).pcd" /> 
            <param name="auto_mode" value="$(arg auto_mode)"/>
            <param name="start_flag" value="$(arg start_flag_lidar)"/>
         </node>
    </group>

    <group ns="camera">
        <node pkg="manual_calibration" name="selectPointCamera" type="selectPointCamera" output="screen">
            <param name="image_filepath" value="$(find manual_calibration)/data/imgFiles/$(arg image_filename).png" />  
            <param name="auto_mode" value="$(arg auto_mode)"/>
            <param name="start_flag" value="$(arg start_flag_cam)"/>
        </node>
    </group>

</launch>

